HRTool/src/main/java/app/controllers/ImageUploadRestController.java

```diff
@@ -14,55 +12,26 @@
 import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.multipart.MultipartFile;
 
-
-
 @Controller
-
 public class ImageUploadRestController {
 
 	@Autowired
-    private ServletContext servletContext;
-
-	
-	@RequestMapping(value = "/upload", method = RequestMethod.GET)
-	public @ResponseBody String uploading(){
-		
-		
-		System.out.println("---------------------------------");
-		return "BLA BLA";
+	private ServletContext servletContext;
 
-
-	}
-	
 	@RequestMapping(value = "/upload", method = RequestMethod.POST)
```

Comment:
RequestMapping URL should target resources, never actions/methods.
Ex. Uploading an image for an employee would target /employees/{employeeId}/image.